src/main/java/seedu/saveit/logic/commands/RedoCommand.java

```diff
@@ -27,6 +30,8 @@ public CommandResult execute(Model model, CommandHistory history) throws Command
 
         model.redoSaveIt();
         model.updateFilteredIssueList(Model.PREDICATE_SHOW_ALL_ISSUES);
+        model.resetDirectory(new Directory(0, 0));
+        EventsCenter.getInstance().post(new DirectoryChangedEvent(new Directory(0, 0)));
```

Comment:
the int number can be changed to a static field

Reply:
Now replaced with a root directory builder